src/nomad_material_processing/__init__.py

```diff
@@ -177,6 +446,15 @@ class Dopant(ElementalComposition):
         ),
         unit='1 / m ** 3',
     )
+    doping_deviation = Quantity(
+        type=float,
+        description='The ± uncertainty in the doping level.',
```

Comment:
If we call the name `_deviation` I would use "deviation" in the description as well.
```suggestion
        description='The ± deviation in the doping level.',
```
Do we need to specify the deviation from what?